src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Emitters/EndpointParameterEmitter.cs

```diff
@@ -44,12 +52,36 @@ internal static void EmitQueryOrHeaderParameterPreparation(this EndpointParamete
 
     internal static void EmitParsingBlock(this EndpointParameter endpointParameter, CodeWriter codeWriter)
     {
-        if (endpointParameter.IsParsable)
+        if (endpointParameter.IsArray && endpointParameter.IsParsable)
+        {
+            codeWriter.WriteLine($"{endpointParameter.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)} {endpointParameter.EmitHandlerArgument()} = new {endpointParameter.ElementType.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)}[{endpointParameter.EmitTempArgument()}.Length];");
+            codeWriter.WriteLine($"for (var i = 0; i < {endpointParameter.EmitTempArgument()}.Length; i++)");
+            codeWriter.StartBlock();
+            codeWriter.WriteLine($"var element = {endpointParameter.EmitTempArgument()}[i];");
+            endpointParameter.ParsingBlockEmitter(codeWriter, "element", "parsed_element");
+
+            // In cases where we are dealing with an array of parsable nullables we need to substitute
+            // empty strings for null values.
+            if (endpointParameter.ElementType.NullableAnnotation == NullableAnnotation.Annotated)
+            {
+                codeWriter.WriteLine($"{endpointParameter.EmitHandlerArgument()}[i] = string.IsNullOrEmpty(element) ? null! : parsed_element!;");
```

Comment:
```suggestion
                codeWriter.WriteLine($"{endpointParameter.EmitHandlerArgument()}[i] = string.IsNullOrEmpty(element) ? null! : parsed_element!;");
```

`null!`?

Reply:
IIRC this is to deal with this situation:

```https://<server>/route?foo=xyz&foo=&foo=abc```

You want the second ```foo``` to be represented as a null, not an empty string according to existing RDF tests).